### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: app/cafe.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
```
Comment (vsmutok) [inline app/cafe.py:5]:
```suggestion
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```

**Review by vsmutok — CHANGES_REQUESTED** [review]
Great 👍

File: app/main.py [hunk 2]
```diff
@@ -1 +1,17 @@
...
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    without_masks = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            without_masks += 1
+    if without_masks != 0:
```
Comment (viktoria-rybenchuk) [inline app/main.py:14]: Don't compare a variable to 0 if there is no need

File: app/cafe.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Person must be vaccinated")
+        if (
+                datetime.datetime.now().date()
```
Comment (viktoria-rybenchuk) [inline app/cafe.py:18]:
```suggestion
               datetime.date.today()
```

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/cafe.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Person must be vaccinated")
+        if (
+                datetime.date.today()
+        ) > visitor.get("vaccine").get("expiration_date"):
```
Comment (pavlejviki) [inline app/cafe.py:19]: condition is readable on a single line, there's no need to format  it this way

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
Well done!

**Pull request closed without merging**